Sources/Classes/core/View/MiniAppViewHandler.swift

```diff
@@ -342,6 +378,19 @@ class MiniAppViewHandler: NSObject {
     func loadWebView(url: URL) {
         // load
     }
+
+    func isValidMiniAppInfo(versionId: String) -> Bool {
+        return !appId.isEmpty || !versionId.isEmpty
```

Comment:
will it be a`||` operation or `&&`, because we need both  appId or versionId to get the path of the bundle.